modules/deleting-the-cluster-logging-operator-by-using-the-cli.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+// * uninstalling/uninstalling-logging.adoc
+
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-07-05
+:_mod-docs-content-type: PROCEDURE
+
+[id="deleting-the-cluster-logging-operator-by-using-the-cli_{context}"]
+= Deleting the {clo} by using the CLI
+
+Delete the {clo} by using the {oc-first}.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You installed the {oc-first}.
+
+.Procedure
+
+. Delete the `ClusterLogForwarder` custom resource (CR) by running the following command:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc delete ClusterLogForwarder __<name>__ -n openshift-logging
+----
+
+. Uninstall the {clo} by running the following command:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc delete OperatorGroup __<name>__ -n openshift-logging
+----
+
+. Delete the subsciption associated with the {clo} by running the following command:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc delete Subscription cluster-logging -n openshift-logging
```

Comment:
Here it's better if we detail a bit more the resource as there are other resources called `Subscription`
```suggestion
$ oc delete subscriptions.operators.coreos.com cluster-logging -n openshift-logging
```